Allow multiple visualizations per page, fixes #32

bmabey opened the pull request:
You can see the updated fix in action here: http://benmabey.com/LDAvis

Included in this PR is another update that allows you to pass the actual data to the visualization object instead of a URI. This change was needed for the IPython integration but is generally useful since it allows you to have a self-contained visualization page that you can send to people.

bmabey commented:
@cpsievert did you see this?

cpsievert commented:
Yep, thank you! Hopefully I'll get to it over the weekend! @kshirley might want to look this over as well.

cpsievert commented on inst/htmljs/ldavis.js:
```diff
@@ -1,36 +1,42 @@
+/* MIT Licence */
+
+'use strict';
+
```
@bmabey I'll admit, I didn't know about "use strict" until now, but it seems like a good idea!

@kshirley In case you're curious, I thought this post does a great job of explaining it -> http://ejohn.org/blog/ecmascript-5-strict-mode-json-and-more/

cpsievert commented on inst/htmljs/ldavis.js:
```diff
@@ -139,8 +151,10 @@ LDAvis = function(to_select, json_file) {
+        console.log('lambda_select', lambda_select);
         d3.select(lambda_select)
             .on("mouseup", function() {
+                console.log('lambda_select mouseup');
```
@bmabey could we get rid of `console.log()`s?

cpsievert commented on inst/htmljs/ldavis.js:
```diff
@@ -425,63 +439,63 @@ LDAvis = function(to_select, json_file) {
-            .range([0, barwidth])
-            .nice();
+                .domain([1, d3.max(barDefault2, function(d) {
+                    return d.Total;
```
I think the lower bound of the domain should remain at 0; otherwise, we run into the problem reported in #33

bmabey commented:
@cpsievert I've removed the `console` calls and fixed the regression for #33. The fixes have been squashed to the same commit. It should be ready to merge in now.

dselivanov commented:
@cpsievert @kshirley how about this PR? Would be very nice to have it merged. I had problems with multiple plots on a single page.

cpsievert commented:
> I had problems with multiple plots on a single page.

You could embed them via an `<iframe>`
